### ISSUE TITLE
ErrorException flock()

### ISSUE THREAD

**Issue by mberneis** [itsgoingd/clockwork#382]
I am getting from time to time the following Exception - After a browser refresh, it disappears.
```
ErrorException
flock(): supplied resource is not a valid stream resource

vendor/itsgoingd/clockwork/Clockwork/Storage/FileStorage.php:185
```
Otherwise, the extension seems to be working ok

**Comment by itsgoingd** [comment]
Hey, is this with the latest version? This should be fixed in 4.0.17, but I might have missed something.

**Comment by mberneis** [comment]
Ahh - I was running 4.0.15. - Upgrade now to the latest and will close the issue
